src/backend/reactions/reaction.cpp

```diff
@@ -71,10 +71,34 @@ namespace big
 
 		if (notify)
 		{
+
```

Comment:
Unncessary whitespace change.

Reply:
Didn't see VS had an option to auto apply clang on save. Just found it and will be using it moving forward.